index.js

```diff
@@ -322,6 +322,17 @@ fastifyOauth2.EPIC_GAMES_CONFIGURATION = {
   tokenPath: '/epic/oauth/v1/token'
 }
 
+/**
+ * Yandex ID docs https://yandex.ru/dev/id/doc/en/
```

Comment:
Very good. I also thought that the lack of references is bad. Thank you for adding the reference.